static/js/components/ProgramCourse.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+import React from 'react';
+import Popover from 'material-ui/Popover';
+
+
+export default class ProgramCourse extends React.Component {
```

Comment:
I think `ProgramCourse` isn't the best name. `CourseDescription` is unfortunately already taken by a component in the dashboard.

maybe `CourseInfoPopover`? Idk, just something that better captures the functionality that this class implements.